app/assets/stylesheets/main.scss

```diff
@@ -5,6 +5,12 @@
 
 /* Dashboard */
 
+.link_disabled {
+  display: block;
+  pointer-events: none;
+  color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.